tests/core/code-stream/test_code_stream.py

```diff
@@ -203,7 +203,10 @@ def test_new_vs_reference_code_stream_iter(bytecode):
     assert latest.program_counter == reference.program_counter
 
 
-@given(read_len=st.integers(min_value=0), bytecode=st.binary(max_size=128))
+@given(
+    read_len=st.integers(min_value=0, max_value=2048),
```

Comment:
This (and below) certainly requires review. `max_value=pow(2,63)-1` works just fine, too. It's the `pow(2,63)` that gets passed implicitly is what breaks the test.

Possibly an off-by-one issue upstream.

Reply:
This seems fine to me.